main/ertp/api/brand.md

```diff
@@ -1,18 +1,31 @@
 # Brand
-The `brand` identifies the kind of issuer, and has a function to get the alleged name for the kind of asset described. The alleged name (such as 'BTC' or 'moola') is provided by the maker of the issuer and should  not be trusted as accurate.
-
-Every amount created by AmountMath will have the same brand, but recipients cannot use the brand by itself to verify that a purported amount is authentic, since the brand can be reused by a misbehaving issuer.
-
-## AllegedName
-Human-readable string name of a kind of digital asset. The alleged name should
-not be trusted as an accurate depiction, since it is provided by
-the maker of the issuer and could be deceptive.
+The `brand` identifies the kind of assets. A particular `brand` has one-to-one relationships within a
+contract `instance`
+with an `issuer` and a `mint`, and one-to-many relationships with `amountMaths`, `purses` and `payments`.
+Meaning for, say, the Quatloos `brand`:
+- There is only one Quatloos `issuer`.
+- There is only one Quatloos `mint`.
+- All `amounts` created via the Quatloos `amountMath` have a Quatloos `brand`.
+- There can be many local copies of the Quatloos `amountMath`, but those `amountMaths` only work on Quatloos branded `amounts`.
+- There can be any number of Quatloos holding `purses`, but those `purses` can only hold Quatloos.
+- There can be any number of Quatloos holding `payments`, but those `payments` can only hold Quatloos.
+
+But recipients of a 
+purported `amount` cannot use the `brand` by itself to verify its authenticity,
+since the `brand` can be reused by a misbehaving `issuer`
+
+All of these relationships are unchangable; if, say, a `mint` is created that makes new Quatloos, it
+can never create new assets of any other `brand`.`
```

Comment:
That's worth adding, but it's not what I was trying to get across. If a mint makes Xs, it will always make Xs. It can't start making Ys. How about:

"All of these relationships are unchangable; if, say, another `mint` is created, it
can never create Quatloos. It has a different `brand`. And once a Quatloos mint is
created, it can only make Quatloos; it can never change and mint Moola, Simoleans, or anything
other than Quatloos"

Reply:
Sounds good!